### PR TITLE
Update Météo France component documentation

### PR DESCRIPTION
**Pull request by oncleben31**
**Description:**
Documentation update to follow the integration of weather alerts in existing Météo-France component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23128

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/meteo_france.markdown [hunk 1]
```diff
@@ -103,6 +105,16 @@ Possible value for each intervals attributes are:
 - 3 Moderate rain
 - 4 Heavy rain
 
+### {% linkable_title About `weather_alert` sensor %}
+
+<p class='note warning'>
+  The weather alert is available for the metropolitan France.<br/>
```
Comment (klaasnicolaas) [inline source/_components/meteo_france.markdown]:
```suggestion
  The weather alert is available for the metropolitan France.
```

File: source/_components/meteo_france.markdown [hunk 2]
```diff
@@ -103,6 +105,16 @@ Possible value for each intervals attributes are:
...
+</p>
+
+The `weather_alert` sensor value give the current weather alert status for the department linked to the city. Data is retrieve from [Météo-France vigilance website](http://vigilance.meteofrance.com/).
+
+The sensor attributes  give access to each type of alerts and date of the bulletin emitted by Météo-France.
```
Comment (klaasnicolaas) [inline source/_components/meteo_france.markdown]:
```suggestion
The sensor attributes give access to each type of alerts and date of the bulletin emitted by Météo-France.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by oncleben31** [comment]
Thanks for the quick review.

**Review by oncleben31 — COMMENTED** [review]
OK taken into account.

**Comment by oncleben31** [comment]
@klaasnicolaas I'm not sure what I have to do for resolving the red "Changes requested" action in the checks?
I've add a commit with your suggestion. Do I need to do something else?

**Comment by klaasnicolaas** [comment]
@oncleben31 You don't have to do anything, it is waiting for the parent PR to merge.

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by oncleben31** [comment]
Hey @klaasnicolaas. It seems no one is available to review my parent PR. Perhaps it's now too low in the PR list to catch attention. Do you have any means to poke some other dev to see if they can invest some time on it ?

**Comment by klaasnicolaas** [comment]
You just have to wait, reviewers work the list in their own way.

**Comment by oncleben31** [comment]
Thanks for taking the time to answer me. I'm new in this PR game.
I've other ideas of improvement for this integration. Do you think I can continue to add features or I should try to keep my PR not too complex

**Comment by klaasnicolaas** [comment]
Depends on what you are planning, but we prefer that PRs do not become too large and complex. So maybe you can do it better in a new PR.

**Review by frenck — APPROVED** [review]

**Pull request merged**